### ISSUE TITLE
gettext commands work but end with DeprecationWarning

### ISSUE THREAD

**Issue by escapedcat** [Polyconseil/easygettext#19]
```
./node_modules/easygettext/dist/extract-cli.js --output *.vue
```
This ends with a _warning_:
```
[easygettext] extracting: 'foo.vue
(node:41795) DeprecationWarning: Calling an asynchronous function without callback is deprecated.
```

`compile-cli.js` also ends with such warning.

Nothing serious, but wanted to mention it.

**Comment by vperron** [comment]
Nice catch. We'll have a look into it (use async instead of synchronous functions) asap. Unless you've got a PR, which would of course be welcome :)

**Comment by escapedcat** [comment]
I actually tried, but somehow didn't know where to start after looking at the code and tests 😕

**Comment by stephane** [comment]
Fixed by a commit of #22
